notebooks/Notebook3.md

```diff
@@ -2,33 +2,66 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+One thing I found especially helpful for my programming this week was suggestions for features.
+For example, some features that were brought up included adding slide numbers, translating elements not off the page, and making it easy to change the order that things show up.
```

Comment:
I really like the transformation that you made that made it possible to move elements to certain areas in the screen!